### DISCUSSION TITLE
dva与react router 6兼容问题

### DISCUSSION THREAD

**Discussion by rimline** [umijs/umi#11318] [Q&A]
dva依赖了connected-react-router，但最新connected-react-router版本不支持react router 6，所以umi4 router 6是怎么和dva兼容的？

**Comment by fz6m** [comment]
dva 很久不维护了，不推荐再使用了，现代数据流更推荐使用 [valtio](https://umijs.org/docs/max/valtio) / [jotai](https://github.com/pmndrs/jotai) 管理全局数据，使用 [react-query](https://umijs.org/docs/max/react-query) 管理请求状态。
